tests/framework/ROM/TimeSeries/ARMA/segmented.xml

```diff
@@ -0,0 +1,129 @@
+<Simulation verbosity="debug">
+  <TestInfo>
+    <name>framework/ROM/TimeSeries/ARMA.Segmented</name>
+    <author>talbpaul</author>
+    <created>2018-11-20</created>
+    <classesTested>SupervisedLearning.ARMA</classesTested>
+    <description>
+      Tests the mechanics of clusterering the "time" subspace of the ARMA ROM.
```

Comment:
any other more info?

Reply:
I feel this succinctly described the behavior, and I'm not sure what additional information could provide further insights to the maintenance of this test. It is exclusively a mechanical test, and there doesn't seem to be anything particularly obscure about it to me.